module/support/optimisation/NSGA2Optimiser/src/nsga2/Individual.cpp

```diff
@@ -0,0 +1,241 @@
+#include "Individual.hpp"
+
+#include <cmath>
+#include <fmt/format.h>
+#include <nuclear>
+
+namespace nsga2 {
+    Individual::Individual(const IndividualConfigurator& _config) : config(_config) {
```

Comment:
I don't think this `_config` variable is ever used, may be a misunderstanding of C++ but what's going on here?